pkg/utils/iptables.go

```diff
@@ -32,15 +33,30 @@ func Restore(table string, data []byte) error {
 	if err != nil {
 		return err
 	}
-	args := []string{"iptables-restore", "-T", table}
+	var args []string
+	args = []string{"iptables-restore", "--help"}
```

Comment:
@murali-reddy Can you think of any easy way that we could just cache this on startup so that we don't have the extra command call for each call to `Restore()`? Maybe make it it's own function and then call it at the beginning of the NPC initialization?

Reply:
Good suggestion. Incorporated.